src/polyfills.ts

```diff
@@ -19,32 +19,32 @@
  */
 
 /** IE9, IE10 and IE11 requires all of the following polyfills. */
-import 'core-js/es6/symbol';
-import 'core-js/es6/object';
-import 'core-js/es6/function';
-import 'core-js/es6/parse-int';
-import 'core-js/es6/parse-float';
-import 'core-js/es6/number';
-import 'core-js/es6/math';
-import 'core-js/es6/string';
-import 'core-js/es6/date';
-import 'core-js/es6/array';
-import 'core-js/es6/regexp';
-import 'core-js/es6/map';
-import 'core-js/es6/weak-map';
-import 'core-js/es6/set';
+// import 'core-js/es/symbol';
+// import 'core-js/es/object';
+// import 'core-js/es/function';
+// import 'core-js/es/parse-int';
+// import 'core-js/es/parse-float';
+// import 'core-js/es/number';
+// import 'core-js/es/math';
+// import 'core-js/es/string';
+// import 'core-js/es/date';
+// import 'core-js/es/array';
+// import 'core-js/es/regexp';
+// import 'core-js/es/map';
+// import 'core-js/es/weak-map';
+// import 'core-js/es/set';
```

Comment:
so what is the plan here, I think we have to at least support ie11 for a little while longer.

Reply:
You can still use the trick in the `tsconfig` to target es5.